### PR TITLE
Update WSL developer documentation to refer to X Server use

### LINKED ISSUE #35030: Update Windows Subsystem for Linux documentation with regards r
**Describe the outcome that is desired.**

**Describe any solutions you are considering**

**Additional context**

### PR DESCRIPTION
**Pull request by MialLewis**
**Description of work.**

This adds instruction for the use of an XServer when using WSL on windows. Without this, the workbench GUI cannot be used.

**To test:**
1) Read through documentation for typos etc.
2) Ideally, follow through instructions to set up and use an XServer with WSL

Resolves #35030 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/WindowsSubsystemForLinux.rst [hunk 1]
```diff
@@ -76,6 +76,30 @@ The Mantid code can then be retrieved in the usual way using `git clone` from th
 
 You are now ready to CMake and build the Mantid code. Follow the Ubuntu 18.04 or Centos 7 build instructions `here <https://developer.mantidproject.org/GettingStarted.html#linux>`_.
 
+Using Graphical User Interfaces (GUI)
+#####################################
+
+WSL does not currently support GUIs. In order to display a GUI from an application running in WSL you will therefore need to use an XServer. Failure to do so will result in a crash upon launching `workbench`.
```
Comment (robertapplin) [inline dev-docs/source/WindowsSubsystemForLinux.rst:82]:
```suggestion
WSL does not currently support GUIs. In order to display a GUI from an application running in WSL you will need to use an XServer. Failure to do so will result in a crash upon launching `workbench`.
```

File: dev-docs/source/WindowsSubsystemForLinux.rst [hunk 2]
```diff
@@ -76,6 +76,30 @@ The Mantid code can then be retrieved in the usual way using `git clone` from th
...
+
+To use an XServer:
+
+1. Download a compatible XServer onto the host Windows machine. `MobaXterm <https://mobaxterm.mobatek.net/>`_ is recommended and has been confirmed to work with Mantid workbench.
```
Comment (robertapplin) [inline dev-docs/source/WindowsSubsystemForLinux.rst:86]:
```suggestion
1. Download a compatible XServer onto the host Windows machine. `MobaXterm <https://mobaxterm.mobatek.net/>`_ is recommended and has been confirmed to work with Mantid Workbench.
```

File: dev-docs/source/WindowsSubsystemForLinux.rst [hunk 3]
```diff
@@ -76,6 +76,30 @@ The Mantid code can then be retrieved in the usual way using `git clone` from th
...
+
+4. Configure the `DISPLAY` variable on WSL. The manner in which this is done differs between WSL1 and WSL2. To check which version you have use the command `wsl -l -v` in the command prompt on the host machine.
+   If you have WSL1, it is recommended to upgrade to `WSL2 <https://learn.microsoft.com/en-us/windows/wsl/install>`_.
+
+  1. On WSL, use `cd` to navigate to your home directory containing the `.bashrc` file.
+  2. Using `vi .bashrc` append the following to the file contents:
+
+     `export DISPLAY=$(route.exe print | grep 0.0.0.0 | head -1 | awk '{print $4}'):0.0`.
+  3. Exit and re-enter WSL via the command prompt.
+  4. Echo the `DISPLAY` variable, you should see the value `<host machine IP Address>:0.0`.
```
Comment (robertapplin) [inline dev-docs/source/WindowsSubsystemForLinux.rst:101]: Would it be useful to say that they can check the IP address here matches the `IPv4 address` displayed when you run `ipconfig` in your terminal? Just as an extra check

**Review by robertapplin — CHANGES_REQUESTED** [review]
Looks good, just had a few minor comments and a question

**Comment by MialLewis** [comment]
> Looks good, just had a few minor comments and a question

Thanks for the feedback, I've actioned all of the comments.

**Review by robertapplin — APPROVED** [review]
Thanks for the changes, looks good! The test failures are unrelated

**Comment by martyngigg** [comment]
Unrelated status check failure.

**Pull request merged**